sdk-extensions/async-processor/src/main/java/io/opentelemetry/sdk/extension/trace/export/DisruptorEventQueue.java

```diff
@@ -205,11 +192,12 @@ public void onEvent(final DisruptorEvent event, long sequence, boolean endOfBatc
           case ON_START:
             @SuppressWarnings("unchecked")
             final SimpleImmutableEntry<ReadWriteSpan, Context> eventArgs =
-                (SimpleImmutableEntry<ReadWriteSpan, Context>) readableSpan;
+                (SimpleImmutableEntry<ReadWriteSpan, Context>)
```

Comment:
These requireNonNull are safe based on the code usage. Presumably we're supposed to have actually different event types for the different arguments. My suggestion is to delete this artifact instead if it seems unused :P